Bottom: 3264e900a8fb97285867055b6f7383f0eefdb19b
Top:    8ef4faddac2b240d168f243782e46cead8025572
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-01 11:44:06 -0700

Refresh of sbdebugger-setinputfile

---
diff --git a/lldb/include/lldb/API/SBFile.h b/lldb/include/lldb/API/SBFile.h
index b3f9b324fca..a5fc2cd4121 100644
--- a/lldb/include/lldb/API/SBFile.h
+++ b/lldb/include/lldb/API/SBFile.h
@@ -33,7 +33,7 @@ public:
 
 private:
   FileSP m_opaque_sp;
-  SBFile(FileSP file_sp) : m_opaque_sp(file_sp) {}
+  SBFile(FileSP file_sp);
 };
 
 } // namespace lldb
diff --git a/lldb/source/API/SBFile.cpp b/lldb/source/API/SBFile.cpp
index fc4f5572f01..b36dedb628f 100644
--- a/lldb/source/API/SBFile.cpp
+++ b/lldb/source/API/SBFile.cpp
@@ -16,6 +16,8 @@ using namespace lldb_private;
 
 SBFile::~SBFile() {}
 
+SBFile::SBFile(FileSP file_sp) : m_opaque_sp(file_sp) {}
+
 SBFile::SBFile() { LLDB_RECORD_CONSTRUCTOR_NO_ARGS(SBFile); }
 
 SBFile::SBFile(FILE *file, bool transfer_ownership) {
